packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -905,4 +961,33 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 			? resolvedPluginConfig.workers[environmentName]
 			: undefined;
 	}
+
+	/**
+	 * @returns Container options suitable for building or pulling images,
+	 * with image tag set to well-known dev format, or undefined if
+	 * containers are not enabled or not configured.
+	 */
+	async function getContainerOptions(
```

Comment:
this aaalmost duplicates the wrangler counterpart, with the exception that `containerBuildId` needs to be passed into this function, as the vite-plugin does not have access to it via the config, as wrangler does